### ISSUE TITLE
Valid and on Win working chocolatey scripts fail

### ISSUE THREAD

**Issue by Schroedingers-Cat** [PietJankbal/Chocolatey-for-wine#1]
When trying to install some packages from the community server (for example this one https://community.chocolatey.org/packages/compassion), the installation fails due to a syntax error. The error occurs on this line:
```powershell
Out-File -FilePath ($env:ChocolateyPackageFolder + "\uninstall.txt")
```
It's failing due to  a missing `"` after uninstall.txt, which doesn't seem to be true. It's working fine on a Windows 10 installation and it's also passing the community package verification tests which include an error-less run of the installation process. Any idea, what could cause this situation?

**Comment by PietJankbal** [comment]
I think it`s a bug in wine, but I don`t know where exactly it comes from.

If I do "choco install compassion --use-system-powershell -y" and wait a few seconds (it spits out some errors in the console, just wait a bit) it seems to install "some" things

Disclaimer: There are quite some installers that suffer from bugs in wine, so they won`t work or one needs a workaround... Atm i cannot help that :(

**Comment by Schroedingers-Cat** [comment]
> Disclaimer: There are quite some installers that suffer from bugs in wine, so they won`t work or one needs a workaround... 

True. In this instance, the installer from this package works fine when installing it manually/graphically through wine, though.  

> "choco install compassion --use-system-powershell -y"  

Interesting. From the docs:
> --use-system-powershell
>     UseSystemPowerShell - Execute PowerShell using an external process 
>       instead of the built-in PowerShell host. Should only be used when 
>       internal host is failing. Available in 0.9.10+.  

I wonder why it helps in this case.

**Comment by PietJankbal** [comment]
Hi

I updated the installscript to disable builtin powershell host. The installation of compassion should now work 'out' of the box, so I close this issue. Thanks for reporting this
